app/scripts/views/components/barChart/barChart.js

```diff
@@ -287,20 +322,26 @@
         }
       }
       chartInst_.render();
+
+      initTsvDownloadData();
+      initCanvasDownloadData();
```

Comment:
Good point. I'm thinking to add 'initDownloadData' function in generalChart. But don't a good solution to implement it yet.